cmd/emailreport/mail_test.go

```diff
@@ -0,0 +1,32 @@
+/// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/BurntSushi/toml"
+	"log"
+	"testing"
+)
+
+func TestHeaderRFC822(t *testing.T) {
+	var conf Configuration
+
+	if _, err := toml.DecodeFile("example.toml", &conf); err != nil {
+		log.Fatal(err)
+	}
+
+	HeaderRFC822(conf)
```

Comment:
something like this: expectedHeader="XYZ";  if exepetedHeader != returnedHeader then error?